src/views/authenticated/List.tsx

```diff
@@ -11,6 +11,7 @@ interface Props {
 
 export function List({ data: unfilteredListItems, listPath }: Props) {
 	const navigate = useNavigate();
+	const { listName } = useParams<{ listName: string }>();
```

Comment:
praise: I like seeing useParams here. It is something that I've used very little so it i nice to see it used here and getting the use that you wanted in it! 🎉

Reply:
+1

Reply:
I was very excited to learn about its existance when I was exploring routing concepts that were more flexible! Thanks for the appreciation!